source/openapi3/split.yaml

```
@@ -2361,6 +2361,43 @@ paths:
           description: Bad Request (errors)
         '404':
           description: Not Found
+  '/contacts/{contact_id}/receivable/':
```

Do we need the trailing slash?
```suggestion
  '/contacts/{contact_id}/receivable':
```